packages/adelus/src/CMakeLists.txt

```diff
@@ -0,0 +1,185 @@
+
+INCLUDE(CheckIncludeFileCXX)
+
+#
+# A) Package-specific configuration options
+#
+
+CHECK_INCLUDE_FILE_CXX(sys/time.h HAVE_SYS_TIME_H)
+CHECK_INCLUDE_FILE_CXX(time.h HAVE_TIME_H)
+
+TRIBITS_CONFIGURE_FILE(${PACKAGE_NAME}_config.h)
+
+#
+# B) Define the header and source files (and directories)
+#
+
+#
+# src
+#
+
+SET(HEADERS "")
+SET(SOURCES "")
+
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
```

Comment:
if possible, consider using target_include_directories instead.  That way downstream libraries will inherit the directories.

Reply:
Thanks @SAdamBlake. I just followed what Belos and Kokkos Kernels do. I will leave it this way. If necessary in the future, I can change it to target_include_directories.